manifests/g/gdiObjects/xBar/2.0.2201.21/gdiObjects.xBar.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: gdiObjects.xBar
+PackageVersion: 2.0.2201.21
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```
https://github.com/microsoft/winget-pkgs/blob/f495a2f541adcb211b7c921b10913a824e674930/manifests/g/gdiObjects/xBar/2.0.2112.20/gdiObjects.xBar.installer.yaml#L7

---

The installer is x86.  xBar is x64.  So it really depends on the meaning of 'Architecture'

---

It depends on the latter; as in application architecture instead of installer architecture.

See https://github.com/microsoft/winget-create/issues/198 for the bug.

---

Ok, how do i approve the change to x64?